--- yrmcds-1.1.8.org/etc/yrmcds.service	2016-10-04 12:39:59.204551985 +0900
+++ yrmcds-1.1.8/etc/yrmcds.service	2016-10-04 12:41:24.896548825 +0900
@@ -1,8 +1,8 @@
 # systemd service for yrmcds
 [Unit]
 Description=yrmcds server
-Wants=networking.target
-After=networking.target
+Wants=network.target
+After=network.target
 
 [Service]
 Type=simple
@@ -11,7 +11,7 @@
 UMask=0077
 PrivateTmp=true
 ExecStartPre=/bin/mkdir -p /var/tmp/yrmcds
-ExecStart=/usr/local/sbin/yrmcdsd
+ExecStart=/usr/sbin/yrmcdsd -f /etc/yrmcds/yrmcds.conf
 Restart=on-failure
 
 [Install]
